collect_app/build.gradle

```diff
@@ -194,12 +196,6 @@ configurations.all {
 }
 
 dependencies {
-    // To make ODK Collect is installable on all devices even though the Mapbox
-    // native library is missing for some ABIs, we include an .aar dependency
-    // named "nbi-stubs", which causes there to be at least one .so library
-    // present for every ABI (armeabi, armeabi-v7a, arm64-v8a, x86, x86_64).
-    implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
-
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
     implementation 'androidx.browser:browser:1.2.0'
     implementation 'androidx.appcompat:appcompat:1.1.0-rc01' // Shouldn't be upgraded until https://issuetracker.google.com/issues/141869006 is fixed
```

Comment:
It's marked as `won't fix`. We can leave it as if for now but in this case we can't block it for ever.

Reply:
We actully have our own issue for tracking this with notes on the possible workaround. I'll update this so it references that rather than Google's issue.